app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,23 @@
+class CommentsController < ApplicationController
+  def new
+    # post_id = params[:id]
+    comment = Comment.new
+    respond_to do |format|
+      # format.html { render :new, locals: { comment: comment, post_id: post_id } }
+      format.html { render :new, locals: { comment: comment } }
+    end
+  end
+
+  def create
+    # new object from params
+    post = Post.find(params[:id])
+    comment = Comment.new(text: params[:comment][:text], post: post, author: current_user)
+    # respond_to block
```

Comment:
- [ ] An error occurs while trying to create a comment for a post. Kindly view the image attached below to see the error 👇 
![image](https://user-images.githubusercontent.com/46839548/201505537-45931b7c-23bd-4a38-a048-1c4237cb1761.png)

Reply:
Done! Thanks